src/community/oseo/oseo-service/src/main/resources/org/geoserver/opensearch/eo/response/generic-content-start.ftl

```diff
@@ -0,0 +1,16 @@
+<#assign a = model.attributes />
+<entry>
```

Comment:
While the product and collection bits might be similar in the current output, the user might just want for them to be different once customized. I'd go for a simpler setup where you have _one_ content template for collection, and one for product, with a little duplication.  So instead of "start"-"product"-"end", just have "product" with everything inside, and the same for collection.

Reply:
Removed content-start and content-end files. There is only a single template for collection and one for product.